File: reference.vb.net/Db4objects.Db4odoc/Messaging/MyClientServerMessage.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 
Imports System

Namespace Db4objects.Db4odoc.Messaging
    Class MyClientServerMessage
        Private _info As String

        Public Sub New(ByVal info As String)
            Me._info = info
        End Sub

        Public Overrides Function ToString() As String
            Return "MyClientServerMessage: " + _info
        End Function
    End Class
End Namespace

File: reference.net/db4o-reference-chapters/src/Db4objects.Db4odoc/Messaging/MyClientServerMessage.cs
using System;

namespace Db4objects.Db4odoc.Messaging
{
	class MyClientServerMessage 
	{
		private string _info;

		public MyClientServerMessage(String info)
		{
			this._info = info;
		}
    
		public override String ToString()
		{
			return "MyClientServerMessage: " + _info;
		}
	}
}
